src/style.css

```diff
@@ -0,0 +1,89 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("./fonts/Avenir.ttc") format("ttc");
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "Avenir-Boock";
+  src: url("./fonts/Avenir-Book.ttf") format("truetype");
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url("./fonts/Avenir-Heavy.ttf") format("truetype");
+  font-weight: 900;
+  font-style: normal;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.search {
+  margin-top: 20px;
+  position: relative;
+
+}
+
+.search-icon {
+  position: absolute;
+  height: 19px;
+  width: 19px;
+  top: 25px;
+  left: 26px;
+}
+
+.small-icon {
+  height: 11px;
+  width: 11px;
+  top: 15px;
+  left: 13px;
+}
+
+.input {
+  height: 70px;
+  width: 100%;
+  font-family: "Avenir-Heavy", serif;
+  font-size: 16px;
+  font-weight: 900;
+  line-height: 22px;
+  color: #3d4e61;
+  padding-left: 62px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
```

Comment:
you may [group tags](https://rehansaeed.com/system-for-grouping-and-sorting-css-properties/), to make it more readable and pretty 😉

Reply:
I've done it, but I'm not sure if that's the way it should be.